playout/libretime_playout/player/schedule.py

```diff
@@ -34,11 +34,11 @@ def insert_event(events: Events, event_key: str, event: AnyEvent) -> None:
     events[key] = event
 
 
-def get_schedule(api_client: ApiClient) -> Events:
+def get_schedule(api_client: ApiClient, cache_ahead_hours: int) -> Events:
     stream_preferences = StreamPreferences(**api_client.get_stream_preferences().json())
 
     current_time = datetime.utcnow()
-    end_time = current_time + timedelta(days=1)
```

Comment:
I would agree, except for the fact that it was incorrectly stating that the default was 1, and the default has really been 24 hours for quite some time. So the default behavior is not changed.

See here for the old behavior. 1 day = 24 hours.